### PR TITLE
Add drone and code checking settings

### PR DESCRIPTION
**Pull request by sbendjebla**
* **Please check if the PR fulfills these requirements**
- [ ] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug Fix
- [ ] New feature
- [ ] Docs

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- [ ] Yes
- [ ] No

* **Other information**:

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -30,6 +47,21 @@ steps:
...
         from_secret: pypi_password
 
+- name: upload_doc_master
+  image: appleboy/drone-scp
+  when:
+    branch: master
+    event: push
+  settings:
+    host: austerlitz.dessia.tech
+    target: /var/www/documentation/volmdlr/
```
Comment (masfaraud) [inline .drone.yml]: change to volmldr -> finite elements

**Review by masfaraud — CHANGES_REQUESTED** [review]

File: .drone.yml [hunk 2]
```diff
@@ -4,20 +4,37 @@ type: docker
...
-- name: develop
-  image: python:3.7
+- name: install, build doc run scripts
+  image: python:3.8
   commands:
-  - python setup.py develop
+  - git fetch --tags
+  - pip install Cython sphinx sphinx_rtd_theme coverage
+  - python setup.py install
+  - cd scripts
```
Comment (masfaraud) [inline .drone.yml:27]: There is no scripts in git. Forgot to add some files to git?

File: .drone.yml (hunk 1 above)
Comment (sbendjebla) [reply .drone.yml]: done !

File: .drone.yml (hunk 2 above)
Comment (sbendjebla) [reply .drone.yml:27]: Actually, I added ci_scripts and some scripts files in the next PR #2

**Review by masfaraud — APPROVED** [review]

**Pull request merged**